src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -1014,6 +1014,27 @@ public void testFailOverOrgAccount() throws SQLException {
     }
   }
 
+  /** Test production connectivity with disableOCSPChecksMode enabled. */
+  @Test
+  public void testDisableOCSPChecksMode() throws SQLException {
```

Comment:
Can we have more tests on various combinations of `disableOCSPChecks` and `insecureMode`? Or maybe this is not a good place, it should be checked in a place where connection string is parsed?